sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -302,28 +303,27 @@ var data = new
 
 Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-JsonElement conversationalTaskResult = result.RootElement;
-JsonElement orchestrationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+dynamic conversationalTaskResult = response.Content.ToDynamic();
+dynamic orchestrationPrediction = conversationalTaskResult.result.prediction;
 ```
 
 #### Question Answering prediction
 
 If your conversation was analyzed by Question Answering, it will include an intent - perhaps even the top intent - from which you can retrieve answers:
 
 ```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionQnA
-string respondingProjectName = orchestrationPrediction.GetProperty("topIntent").GetString();
-JsonElement targetIntentResult = orchestrationPrediction.GetProperty("intents").GetProperty(respondingProjectName);
+string respondingProjectName = orchestrationPrediction.topIntent;
+dynamic targetIntentResult = orchestrationPrediction.intents[respondingProjectName];
 
-if (targetIntentResult.GetProperty("targetProjectKind").GetString() == "QuestionAnswering")
+if (targetIntentResult.targetProjectKind == "QuestionAnswering")
 {
     Console.WriteLine($"Top intent: {respondingProjectName}");
 
-    JsonElement questionAnsweringResponse = targetIntentResult.GetProperty("result");
+    dynamic questionAnsweringResponse = targetIntentResult.result;
     Console.WriteLine($"Question Answering Response:");
-    foreach (JsonElement answer in questionAnsweringResponse.GetProperty("answers").EnumerateArray())
+    foreach (dynamic answer in questionAnsweringResponse.answers)
     {
-        Console.WriteLine(answer.GetProperty("answer").GetString());
+        Console.WriteLine(answer.answer?.ToString());
```

Comment:
The semantics of DynamicJson.ToString() and JsonElement.GetString() are different, although the result is the same if the JSON property has a string value.  The latter will throw if the property doesn't have a string value, whereas DynamicJson.ToString() will always return the underlying JSON as a string.

For disambiguating the type to call Console.WriteLine(), the intended approach is either to use a cast to the expected type:
```csharp
Console.WriteLine((string)answer.answer);
``
Or an interpolated string, as illustrated in the generator samples: https://github.com/Azure/autorest.csharp/issues/3192#issue-1612375261

Reply:
Thinking about this more, I'd rather leave `ToString`. All the interpolated strings above are calling `ToString` so this is consistent and, I bet, will be more common.

Reply:
Yes, happy to document that - good call out.

Sorry, no, I must have misspoken.  GetString() is not a valid call on DynamicJson - I only mentioned it because in the original code that you are changing, you had called it on the JsonElement.

Since these are samples that customers may copy, I think it would be better if we show the intended usage. I see your point that ToString() works here in a call to Console.WriteLine(), but I'd be concerned people might confuse it with the goal of GetString().  Do you think that is valid?  Maybe this isn't a concern.